plaid/environments.go

```diff
@@ -1,12 +1,20 @@
 package plaid
 
-type Environment string
+type Environment struct {
+	URL string
```

Comment:
nit: can we make `url` unexported and implement a `Getter` on the environment? I don't like the idea of a caller being able to potentially mutate these now that they are variables?